index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
```

Comment:
You want `addonContext.parent.registry` (`addonContext.app` is not always available).